src/components/submission/list.vue

```diff
@@ -29,19 +29,29 @@ except according to the terms contained in the LICENSE file.
             v-model:submissionDate="submissionDateRange"
             v-model:reviewState="reviewStates"
             :disabled="deleted" :disabled-message="deleted ? $t('filterDisabledMessage') : null"/>
+        </form>
+        <form v-if="!draft" class="form-inline field-dropdown-form" @submit.prevent>
```

Comment:
is there a reason the v-if is on the `form` here and on teh `submission-filters` above?

Reply:
I have moved v-if to the form element of `submission-filters`, ideally there should be just one `form` here